foundationdb/examples/class-scheduling.rs

```diff
@@ -150,32 +180,40 @@ async fn ditch_trx(trx: &Transaction, student: &str, class: &str) {
     //println!("{} ditching class: {}", student, class);
     trx.set(&class_key, &pack(&available_seats));
     trx.clear(&attends_key);
+    
+    Ok(true)
 }
 
-async fn ditch(db: &Database, student: String, class: String) -> Result<()> {
-    db.transact_boxed_local(
+async fn ditch(db: &Database, student: String, class: String) -> Result<bool> {
+    db.transact_boxed(
```

Comment:
Could you remove the transact_boxed and use `db.run`, to make it more like the [java bindings](https://apple.github.io/foundationdb/class-scheduling-java.html#appendix-classscheduling-java)?